apps/glusterfs/placer_arbiter.go

```diff
@@ -234,7 +256,7 @@ func (bp *ArbiterBrickPlacer) tryPlaceBrickOnDevice(
 
 	logger.Debug("Trying to place brick on device %v", device.Info.Id)
 
-	for i, b := range bs.Bricks {
+	for i, b := range bs.Contents() {
```

Comment:
Contents() is not a newly introduced function. Why the change?

Reply:
Contents was introduced in the previous change. It returns a copy of the Bricks with all nulls removed for easy iteration.